sapi/cli/php_cli.c

```diff
@@ -1208,6 +1212,11 @@ int main(int argc, char *argv[])
 	 */
 	argv = save_ps_args(argc, argv);
 
+#ifdef PHP_WIN32
```

Comment:
> This also needs an #ifndef PHP_CLI_WIN32_NO_CONSOLE guard.

I'm don't know PHP internals that much, but a 3rd party app could attach a console to a php.exe compiled with PHP_CLI_WIN32_NO_CONSOLE?